flixel/util/FlxPath.hx

```diff
@@ -207,12 +191,21 @@ class FlxPath implements IFlxDestroyable
 	
 	/**
 	 * Internal function for moving the object along the path.
-	 * Generally this function is called automatically by preUpdate().
 	 * The first half of the function decides if the object can advance to the next node in the path,
 	 * while the second half handles actually picking a velocity toward the next node.
 	 */
 	public function update(elapsed:Float):Void
 	{
+		if (object == null)
+			return;
+		
+		if (_firstUpdate)
```

Comment:
This is a terrible hack, but I couldn't think of a better solution right off the bat now that `start()` doesn't have an `Object` parameter anymore.